tests/backstop_data/engine_scripts/puppet/mosaico/mailing/editor-test.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+module.exports = async (engine, scenario, viewport) => {
+  await require('./step-2.js') (engine, scenario, viewport);
+  await engine.waitFor('a[title="Edit"]');
+  // wait for Angular to bind function
+  await engine.waitFor(200);
+  await engine.click('a[title="Edit"]');
+  await engine.waitFor('.status-start', { hidden: true });
+  await engine.waitFor(() => document.querySelectorAll('body> iframe.ui-front').length > 0);
```

Comment:
what do you need this for?

Reply:
To make sure that before pupetter counts the frames, the iframe we are looking for is ready and loaded.